index.html

```diff
@@ -225,6 +225,57 @@ <h4>Install the client on your headset</h4>
+                                    <img src="screenshots/settings/bitrate.jpg" class="d-block w-100">
+                                </div>
+                                <div class="tab-pane fade" id="encoders" role="tabpanel" aria-labelledby="encoders">
+                                    <p>Encoders compresses video data in real-time to reduce bandwidth and latency, enabling smooth transmission to the headset.</p>
+                                    <p>Default value: 3 encoders in order 1/8th, 3/8th and 4/8th of the image.</p>
+                                    <p>WiVRn has the ability to split the video in blocks that are processed independently, this may use resources more effectively and reduce latency. All the provided encoders are put into groups, groups are executed concurrently and items within a group are processed sequentially.</p>
+                                    <p>It is possible to mix CPU/GPU encoding if you have spare CPU cores. On AMD systems, this can provide a significant improvement. For high bitrates, avoid software encoding, as CPU time increases significantly.</p>
+                                    <h5><strong>Encoder</strong></h5>
+                                    <p>Default value: <mark>nvenc</mark> if Nvidia GPU and compiled with nvenc, <mark>vaapi</mark> for all other GPU when compiled with ffmpeg, else <mark>x264</mark>.</p>
+                                    <p>Identifier of the encoder, one of</p>
+                                    <ul>
+                                        <li><mark>x264:</mark> software encoding</li>
+                                        <li><mark>NVENC:</mark>  Nvidia hardware encoding</li>
+                                        <li><mark>VAAPI:</mark>  AMD/Intel hardware encoding</li>
+                                        <li><mark>Vulkan:</mark>  experimental, for any GPU that supports Vulkan video encode</li>
+                                    </ul>
+                                    <h5><strong>Codec</strong></h5>
+                                    <p>Default value: first supported by both headset and encoder of <mark>av1</mark>, <mark>h264</mark>, <mark>h265</mark>.</p>
+                                    <p>One of <mark>h264</mark>, <mark>h265</mark> or <mark>av1</mark>. Not all encoders support every codec, <mark>x264</mark> and <mark>vulkan</mark> only support <mark>h264</mark>, <mark>nvenc</mark> only <mark>h264</mark> and <mark>h265</mark>.</p>
+                                    <p>On the dashboard, you can add a new encoder by splitting an existing one and clicking near an edge. Drag the edge to resize, or remove encoders by moving the line to one of the limits.</p>
+                                    <p>By clicking on each block, like the one highlighted in blue in the image, you can select the Encoder and Codec.</p>
+                                    <img src="screenshots/settings/encoders.jpg" class="d-block w-100">
```

Comment:
I see that I didn't describe one element.

The image that we encode is on left and right eye side by side. If you split encoders right in the middle, this provides a little performance improvement when decoding the image.

That's why default is to use the upper left corner (half width, 1/4 height), bottom left (half width, 3/4 height) and right (half width, full height).
Adding this in the text may be overwhelming, but it would be nice to show that partitioning in the screenshot